x/axelarnet/keeper/keeper.go

```diff
@@ -145,6 +145,12 @@ func (k Keeper) DeletePendingIBCTransfer(ctx sdk.Context, portID, channelID stri
 	k.getStore(ctx).Delete(key)
 }
 
+// IsCosmosChain returns true if the given chain name is for a cosmos chain
+func (k Keeper) IsCosmosChain(ctx sdk.Context, chain string) bool {
+	key := cosmosChainPrefix.Append(utils.LowerCaseKey(chain))
```

Comment:
why not reuse `getCosmosChain`?

Reply:
Because the `CosmosChain` struct is part of the `types` package instead of `exported`. If we were to re-use that function, we would need to import the `types` package of a different module, whereas the best practice in this scenario is to import the `exported` package. An alternative would be to move `CosmosChain`  to `exported`, but that would require a larger refactoring.

Reply:
i mean sth like this
```suggestion
	_, ok := k.getCosmosChain(ctx, chain)
	return ok
```

Reply:
Ahh I see.yeah, I guess that could work too